### ISSUE TITLE
Add option to pass number of workers to torch dataloaders

### ISSUE THREAD

**Issue by RaczeQ** [kraina-ai/srai#456]
Currently dataloaders don't use any kwargs and work in single-threaded mode. Add option to use not only `**trainer_kwarg`s but also `**dataloader_kwargs`.

**Comment by simonusher** [comment]
Hi @sabman! First of all, thank you for working on this.

As for maintaining backwards compatibility, it hasn't been our top priority, but we haven't fully ignored it either. I think we might have had at least a couple of breaking changes on minor versions as "this is new, we're moving fast and things may break". Mostly because we didn't know how people would use the library (and they'd want to use it), and if the API we had designed made sense.
But if you're willing to add a deprecation warning then we'd appreciate this and we can maintain it up to the next major. Imo it's not a huge change, so we can do without it.

So in short: if you feel it's worth to add a deprecation warning then go for it, we'll appreciate it. If not then we can release the next minor with it, not fully following semantic versioning yet.
